src/PsrTarget.php

```diff
@@ -39,6 +39,22 @@ class PsrTarget extends Target implements LoggerAwareInterface
         LogLevel::DEBUG => LogLevel::DEBUG,
     ];
 
+    /**
+     * @var array
+     */
+    private $_interestedLevels = null;
+
+    /**
+     * @inheritdoc
+     */
+    public function init()
+    {
+        parent::init();
+        if (is_null($this->_interestedLevels)) {
+            $this->_interestedLevels = $this->_psrLevels;
```

Comment:
I think that by default there should be both PSR and non-PSR levels. Would simplify code significantly.

Reply:
I did not quite understand the point. You mean the division into PSR and non-PSR levels?

Reply:
I mean `$this->_interestedLevels = []`

Reply:
In this case, _psrLevels will duplicate _levels. I left only _levels.